packages/devtools_app/lib/src/shared/console/eval/inspector_service.dart

```diff
@@ -15,13 +15,14 @@ import 'package:flutter/cupertino.dart';
 import 'package:flutter/foundation.dart';
```

Comment:
This has a lot of non-console stuff in it. I would suggest either leaving it in the inspector and accepting slightly tangled imports or move it to a shared directory. Seems odd to have this core inspector logic under the console rather than inspector or shared.

Reply:
Addressed in https://github.com/flutter/devtools/pull/5006.